deployment/terraform/coordinator.go

```diff
@@ -187,19 +182,15 @@ func (t *Terraform) StopCoordinator() (coordinator.Status, error) {
 // coordinator in the current load-test deployment.
 func (t *Terraform) GetCoordinatorStatus() (coordinator.Status, error) {
 	var status coordinator.Status
-	if err := t.preFlightCheck(); err != nil {
-		return status, err
-	}
 
-	output, err := t.Output()
-	if err != nil {
+	if _, err := t.Output(); err != nil {
```

Comment:
Since we are in the package itself, I am wondering if we can use an internal method which does not return the config? Because it looks like all we are doing is populating the `t.output` variable. 0/5. It seems a bit odd to return the output, but not use it.

Reply:
Sure we can add an internal method that only sets it if not set already.